### PR TITLE
Initial work on table reworking for method configs

### PR DESCRIPTION
**Pull request by MattPutnam**

### CODE REVIEW

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs [hunk 1]
```diff
@@ -38,60 +38,55 @@
...
+                             2 (cell (conf "namespace"))
+                             3 (cell (conf "root-ent-type"))
+                             4 (cell (str ((conf "workspaceName") "namespace") ":"
+                                       ((conf "workspaceName") "name")))
+                             5 (cell (str ((conf "methodStoreMethod") "methodNamespace") ":"
+                                       ((conf "methodStoreMethod") "methodName") ":"
+                                       ((conf "methodStoreMethod") "methodVersion")))
+                             6 (cell (str (get (conf "methodStoreConfig") "namespace") ":"
+                                       ((conf "methodStoreConfig") "name") ":"
+                                       ((conf "methodStoreConfig") "version")))
```
Comment (rushtong) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs]: The fields of methodStoreConfig are as follows, so these aren't showing up in the UI:
```
    "methodStoreConfig": {
      "methodConfigNamespace": "repo_method_configs",
      "methodConfigName": "repo_method_config1",
      "methodConfigVersion": "2"
    }
```

**Comment by rushtong** [comment]
:+1: I like it!

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs [hunk 2]
```diff
@@ -38,60 +38,55 @@
...
+                             1 (cell [:a {:href "javascript:;"
+                                          :style {:color (:button-blue style/colors)
+                                                  :textDecoration "none"}} (conf "name")])
+                             2 (cell (conf "namespace"))
+                             3 (cell (conf "root-ent-type"))
+                             4 (cell (str ((conf "workspaceName") "namespace") ":"
+                                       ((conf "workspaceName") "name")))
+                             5 (cell (str ((conf "methodStoreMethod") "methodNamespace") ":"
+                                       ((conf "methodStoreMethod") "methodName") ":"
+                                       ((conf "methodStoreMethod") "methodVersion")))
```
Comment (rushtong) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs]: There's got to be a better clojure way of joining strings with a colon - this is just painful to read.

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs (hunk 1 above)
Comment (eddiebroad) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs]: that's under the "Method" column I believe, right Matt?

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs (hunk 2 above)
Comment (eddiebroad) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs]: now the "config" column just shows ":::"....is that right?

**Pull request merged**